zc_plugins/EmailArchiveManager/v4.0.0/admin/email_archive_manager.php

```diff
@@ -376,14 +406,14 @@ function zen_is_message_trustable($from, $module): bool
             <div class="col-sm-4">
                 <div class="checkbox">
                     <label for="print_format">
-                        <?= zen_draw_checkbox_field('print_format', 1, $isForDisplay, '', 'id="print_format"') ?>
+                        <?= zen_draw_checkbox_field('print_format', 1, !$isForDisplay, '', 'id="print_format"') ?>
                         <?= HEADING_PRINT_FORMAT ?>
                     </label>
                 </div>
                 <div class="checkbox">
-                    <label for="errors_only">
-                        <?= zen_draw_checkbox_field('errors_only', 1, $errorsOnly, '', 'id="errors_only"') ?>
-                        <?= HEADING_ERRORS_ONLY ?>
+                <label for="print_format">
```

Comment:
I think you meant this to be `for="only_errors"`

Reply:
Correct, I'll push. Not sure how pushing from VS Code will work with this github Review, let's see...